rebar.config

```diff
@@ -55,7 +55,7 @@
     , {replayq, {git, "https://github.com/emqx/replayq", {tag, "0.3.4"}}}
     , {pbkdf2, {git, "https://github.com/emqx/erlang-pbkdf2.git", {branch, "2.0.4"}}}
     , {emqtt, {git, "https://github.com/emqx/emqtt", {tag, "1.2.3.1"}}}
-    , {rulesql, {git, "https://github.com/emqx/rulesql", {tag, "0.1.2"}}}
+    , {rulesql, {git, "https://github.com/emqx/rulesql", {tag, "0.1.4"}}}
```

Comment:
Needed to fix a couple warnings that were due to incorrect typespecs from the dependency.

Diff: https://github.com/emqx/rulesql/compare/0.1.2...0.1.4